electron/achievements.js

```diff
@@ -2,16 +2,22 @@
 const greenworks = require("./greenworks");
 const log = require("electron-log");
 
-function enableAchievementsInterval(window) {
+async function enableAchievementsInterval(window) {
```

Comment:
Yes. There is no need to have the game startup process wait for `enableAchievementsInterval` to finish.

This might change if we want to grant in-game achievements matching Steam achievements, but currently the data flow is only from game to Steam.